xmipp

```diff
@@ -371,7 +374,7 @@ if __name__ == "__main__":
 	allSubparser = subparsers.add_parser(MODE_ALL, formatter_class=ModeHelpFormatter)
 	allSubparser.add_argument(*getParamNames(PARAM_JOBS), type=int, default=JOBS)
 	allSubparser.add_argument(*getParamNames(PARAM_BRANCH))
-	allSubparser.add_argument(*getParamNames(PARAM_PRODUCTION), type=bool, default=False)
```

Comment:
I prefer the previous version but here instead of using `type=bool, default=False` and replace it with `action='store_true'`